aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -61,6 +61,9 @@ Tensor pad_tensor_to_shape(
 
 Tensor NestedTensor_nested_tensor_from_mask(const Tensor& t, const Tensor& mask, bool mask_check) {
     TORCH_CHECK(mask.scalar_type() == at::ScalarType::Bool, "Expected mask to be of ScalarType Bool, but got ", mask.scalar_type(), " instead.");
```

Comment:
seems like this line should be removed :)

Reply:
That's what happens when I try to do things quick.  I agree that we're not calling this today.  So, we don't need to update the check.  (And the change does exactly that, because I added the old check back but had not taken the second check out.  )

The original change was not motivated by us calling this function with float today, but rather with the intent that "we're treating floats as legal type for masks, so let's make them legal wherever masks go".  Maybe a bit over-eager ;)
So, removing that bit and dialing it back to where it was